package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "practice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "dependencies": {
+    "express": "^4.17.1",
+    "express-handlebars": "^4.0.3",
+    "mongoose": "^5.9.9"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "node server.js"
```

Comment:
Gotta make sure you have a package.json file otherwise if you try to clone your repo onto another computer it'll be very difficult to get the project created with the right dependencies since we don't want to be cloning our node_modules folder.

Also I've added a start script here so all you need to do is type `npm start` to start up your app.